### ISSUE TITLE
sendmail build failing on missing /etc/mail/sendmail.cf

### ISSUE THREAD

**Issue by jimbygh** [trueos/trueos#236]
related to closed issue #226, but different.   (in the notes below the hostname is `hpsnap195`)

platform hpelite desktop mini, snapshot #195 with unstable ports.

steps to reproduce:
1. install snapshot
2. download src.txz, ports.txz and  untar
2. configure repo to point to unstable instead of snapshot
3. install llvm60, git, poudriere
4. bootstrap-compiler llvm60
5. cd /usr/src; make buildworld buildkernel

build fails with the following at the end:

```
... 
===> usr.sbin/sendmail/confs (all)
===> usr.sbin/sendmail/mail (all)
cp -f /usr/src/usr.sbin/sendmail/sendmailetc/freebsd.mc hpsnap195.mc
/usr/bin/m4 -D_CF_DIR_=/usr/share/sendmail/cf/   /usr/share/sendmail/cf/m4/cf.m4 hpsnap195.mc > hpsnap195.cf
cp -f /usr/src/usr.sbin/sendmail/sendmailetc/freebsd.submit.mc hpsnap195.submit.mc
/usr/bin/m4 -D_CF_DIR_=/usr/share/sendmail/cf/   /usr/share/sendmail/cf/m4/cf.m4 hpsnap195.submit.mc > hpsnap195.submit.cf
/usr/sbin/sendmail -bi -OAliasFile=/etc/mail/aliases
/etc/mail/sendmail.cf: line 0: cannot open: No such file or directory
*** Error code 72
Stop.
make[5]: stopped in /usr/src/usr.sbin/sendmail/mail
*** Error code 1
```

this problem exists in `/usr/src/usr.sbin/sendmail/mail/Makefile` 
the long commentary introduction notes that SENDMAIL_MC can be set in /etc/make.conf.
however, notes in  make.conf(5) caution against setting that.
in the above Makefile, the INSTALL_CF and INSTALL_SUBMIT_CF variables are not defined, so the `cf`  target in 

```
all:  cf maps aliases
```
does nothing at all.

the issue can be resolved by adjusting the  `all`  target to:

```
all:   install-cf  install-submit-cf  maps  aliases
```

this correctly builds sendmail.cf and submit.cf and installs them in /etc/mail, and the build completes successfully.

i'm not sure of the best fix for trueos, if it can be done in an overlay, or with some additional variables in a jenkins build.

--Jim B.

**Comment by jimbygh** [comment]
update:  with the modification to `/usr/src/usr.sbin/sendmail/mail/Makefile`  above, the `make buildworld makekernel` step completed successfully.

**Comment by revhelix** [comment]
I have encountered this issue as well.

Applying Jim's fix.

**Comment by kmoore134** [comment]
I also just committed a fix in PR #250 which will make sure this file exists on fresh systems.
